hbw/app/javascript/packs/hbw/connection.js

```diff
@@ -216,11 +216,19 @@ modulejs.define('HBWConnection', ['jQuery'], (jQuery) => {
         this.options.host = `${document.location.protocol}//${document.location.host}`;
       }
 
+      this.payload = options.payload;
+
       this.serverURL = this.options.host + this.options.path;
       this.channels = {};
     }
 
     request (...args) {
+      if (typeof args[0].data === 'undefined') {
```

Comment:
https://stackoverflow.com/a/3390635
```
args[0].data === undefined
```
is enough. Let's assume that ```undefined``` is not redefined.